C2DExamples.md

```diff
@@ -0,0 +1,616 @@
+      }
+    ]
+  }
+```
+
+Next, we define the metadata for the Dataset and Algorithm that will describe our data assets. This is what we call the DDOs
+```Typescript
+  const DATASET_DDO: any = {
+    '@context': ['https://w3id.org/did/v1'],
+    id: '',
+    version: '4.1.0',
+    chainId: 4,
+    nftAddress: '0x0',
+    metadata: {
+      created: '2021-12-20T14:35:20Z',
+      updated: '2021-12-20T14:35:20Z',
+      type: 'dataset',
+      name: 'dataset-name',
+      description: 'Ocean protocol test dataset description',
+      author: 'oceanprotocol-team',
+      license: 'MIT'
+    },
+    services: [
+      {
+        id: 'testFakeId',
+        type: 'compute',
+        files: '',
+        datatokenAddress: '0x0',
+        serviceEndpoint: 'https://providerv4.rinkeby.oceanprotocol.com',
+        timeout: 60,
+        compute: {
+          publisherTrustedAlgorithmPublishers: [],
+          publisherTrustedAlgorithms: [],
+          allowRawAlgorithm: true,
+          allowNetworkAccess: true
+        }
+      }
+    ]
+  }
+
+  const ALGORITHM_DDO: any = {
+    '@context': ['https://w3id.org/did/v1'],
+    id: '',
+    version: '4.1.0',
+    chainId: 4,
+    nftAddress: '0x0',
+    metadata: {
+      created: '2021-12-20T14:35:20Z',
+      updated: '2021-12-20T14:35:20Z',
+      type: 'algorithm',
+      name: 'algorithm-name',
+      description: 'Ocean protocol test algorithm description',
+      author: 'oceanprotocol-team',
+      license: 'MIT',
+      algorithm: {
+        language: 'Node.js',
+        version: '1.0.0',
+        container: {
+          entrypoint: 'node $ALGO',
+          image: 'ubuntu',
+          tag: 'latest',
+          checksum: '44e10daa6637893f4276bb8d7301eb35306ece50f61ca34dcab550'
+        }
+      }
+    },
+    services: [
+      {
+        id: 'testFakeId',
+        type: 'access',
+        files: '',
+        datatokenAddress: '0x0',
+        serviceEndpoint: 'https://providerv4.rinkeby.oceanprotocol.com',
+        timeout: 60
+      }
+    ]
+  }
+```
+
+We load the configuration:
+```Typescript
+  
+    config = await getTestConfig(web3)
+    aquarius = new Aquarius(config.metadataCacheUri)
+    providerUrl = config.providerUri
+```
+As we go along it's a good idea to console log the values so that you check they are right
+```Typescript
+    console.log(`Aquarius URL: ${config.metadataCacheUri}`)
+    console.log(`Provider URL: ${providerUrl}`)
+  
+```
+
+## 5. Initialize accounts and deploy contracts
+  ### 5.1 Initialize accounts
+```Typescript
+    const accounts = await web3.eth.getAccounts()
+    publisherAccount = accounts[0]
+    consumerAccount = accounts[1]
+```
+Again, lets console log the values so that we can check that they have been saved properly
+```Typescript
+    console.log(`Publisher account address: ${publisherAccount}`)
+    console.log(`Consumer account address: ${consumerAccount}`)
+  
+```
+
+  ### 5.2 Next, lets get the address of the deployed contracts
```

Comment:
can we please not mix up headings and instructional texts. Suddenly here all the headings are what should be just the explaining copy. Find actual headings for those